tests/snappi/pfc/files/helper.py

```diff
@@ -38,7 +40,8 @@ def run_pfc_test(api,
                  bg_prio_list,
                  prio_dscp_map,
                  test_traffic_pause,
-                 headroom_test_params=None):
+                 headroom_test_params=None,
+                 pfc_pause_zero_src_mac=False):
```

Comment:
How about having a variable named "pfc_src_mac" then setting it to 00:00:00:00:00:00.